sqlnexus/TextRowsets.xml

```diff
@@ -4713,100 +4713,113 @@
-			<Column name="File_System" type="RowsetImportEngine.VarCharColumn" length="12" />
-			<Column name="Block_Size" type="RowsetImportEngine.IntColumn"/>
-			<Column name="Boot_Volume" type="RowsetImportEngine.VarCharColumn" length="12" />
-			<Column name="System_Volume" type="RowsetImportEngine.VarCharColumn" length="14" />
-			<Column name="Disk_Quotas" type="RowsetImportEngine.VarCharColumn" length="12" />
-			<Column name="Compression" type="RowsetImportEngine.VarCharColumn" length="12" />
-			<Column name="Capacity_GB" type="RowsetImportEngine.DecimalColumn" />
-			<Column name="FreeSpace_GB" type="RowsetImportEngine.DecimalColumn" />
-		</KnownColumns>
-	</Rowset>
-	  
+        <Column name="Severity" type="RowsetImportEngine.VarCharColumn" length="15" />
+        <Column name="Message" type="RowsetImportEngine.VarCharColumn" length="1100" />
+        <Column name="HelpURL" type="RowsetImportEngine.VarCharColumn" length="200" />
+      </KnownColumns>
+    </Rowset>
+
+    <Rowset name="tbl_disk_volume_information" enabled="true" identifier="-- Volume_Information --" type="RowsetImportEngine.TextRowset">
+      <KnownColumns>
+        <Column name="Volume_Name" type="RowsetImportEngine.NVarCharColumn" length="50" />
+        <Column name="Volume_Label" type="RowsetImportEngine.NVarCharColumn" length="16" />
+        <Column name="Device_ID" type="RowsetImportEngine.NVarCharColumn" length="50"/>
+        <Column name="File_System" type="RowsetImportEngine.VarCharColumn" length="12" />
+        <Column name="Block_Size" type="RowsetImportEngine.IntColumn"/>
+        <Column name="Boot_Volume" type="RowsetImportEngine.VarCharColumn" length="12" />
+        <Column name="System_Volume" type="RowsetImportEngine.VarCharColumn" length="14" />
+        <Column name="Disk_Quotas" type="RowsetImportEngine.VarCharColumn" length="12" />
+        <Column name="Compression" type="RowsetImportEngine.VarCharColumn" length="12" />
+        <Column name="Capacity_GB" type="RowsetImportEngine.DecimalColumn" />
+        <Column name="FreeSpace_GB" type="RowsetImportEngine.DecimalColumn" />
+      </KnownColumns>
+    </Rowset>
+
+    <Rowset name="tbl_running_drivers" enabled="true" identifier="-- running_drivers --" type="RowsetImportEngine.TextRowset">
+      <KnownColumns>
+        <Column name="ID" type="RowsetImportEngine.IntColumn" length="50" />
+        <Column name="File_Name" type="RowsetImportEngine.VarCharColumn" length="120" />
+        <Column name="Company_Name" type="RowsetImportEngine.VarCharColumn" length="50"/>
+        <Column name="File_Description" type="RowsetImportEngine.VarCharColumn" length="50" />
+        <Column name="Product_Version" type="RowsetImportEngine.VarCharColumn" length="20" />
+        <Column name="File_Version" type="RowsetImportEngine.VarCharColumn" length="25" />
+        <Column name="File_Size_MB" type="RowsetImportEngine.DecimalColumn"/>
```

Comment:
```suggestion
        <Column name="File_Size_MB" type="RowsetImportEngine.VarCharColumn" length="12" />
```

Reply:
if we have numbers with commas as decimal separator, we have this issue. If replace commas by points, then it works

Reply:
@hacitandogan, can you look at this. We resolved this in #444 , but seems some other flavor of it is still present.